RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -4,122 +4,92 @@
-                                                         'drop DetIdfloatstdmap_hgcalLayerClusters_*_*',
-                                                         'keep *_hgcalMultiClusters_*_*',
-                                                         'keep *_iterHGCalMultiClusters_*_*'] )
-# don't modify AOD for HGCal yet, need "reduced" rechits collection first (i.e. requires reconstruction)
-phase2_hgcal.toModify( RecoLocalCaloAOD,
-    outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*',
-                                                        'keep *_hgcalLayerClusters_*_*',
-                                                        'drop DetIdfloatstdmap_hgcalLayerClusters_*_*'])
-
-# mods for HFNOSE
 from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
-
-phase2_hfnose.toModify( RecoLocalCaloRECO,
-                        outputCommands = RecoLocalCaloRECO.outputCommands + ['keep *_hgcalLayerClustersHFNose_*_*',
-                                                                             'drop DetIdfloatstdmap_hgcalLayerClustersHFNose_*_*',
-                                                                          ] )
-
-phase2_hfnose.toModify( RecoLocalCaloFEVT,
-                        outputCommands = RecoLocalCaloFEVT.outputCommands + ['keep *_hgcalLayerClustersHFNose_*_*',
-                                                                             'drop DetIdfloatstdmap_hgcalLayerClustersHFNose_*_*'] )
-
-phase2_hfnose.toModify( RecoLocalCaloAOD,
-                        outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_hgcalLayerClustersHFNose_*_*',
-                                                                            'drop DetIdfloatstdmap_hgcalLayerClustersHFNose_*_*'] )
-
 from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+# don't modify AOD for HGCal yet, need "reduced" rechits collection first (i.e. requires reconstruction)
+phase2_hgcal.toModify( RecoLocalCaloAOD, 
+    outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*',
+                                                        'keep recoCaloClusters_hgcalLayerClusters_*_*',
+                                                        'keep *_hgcalLayerClusters_timeLayerCluster_*',
+                                                        'keep *_hgcalLayerClusters_InitialLayerClustersMask_*'])
+phase2_hfnose.toModify( RecoLocalCaloAOD, 
+    outputCommands = RecoLocalCaloAOD.outputCommands + ['keep recoCaloClusters_hgcalLayerClustersHFNose_*_*',
+                                                        'keep *_hgcalLayerClustersHFNose_timeLayerCluster_*',
+                                                        'keep *_hgcalLayerClustersHFNose_InitialLayerClustersMask_*'])
 (pA_2016|pp_on_AA_2018).toModify( RecoLocalCaloAOD.outputCommands,
```

Comment:
Just noticing that in the previous design the customizations for HI and LowPt only added products to the AOD event content. Now you correctly propagate them also to the RECO and FEVT event contents, as it shoudl